view/templates/item/compose.tpl

```diff
@@ -81,6 +81,8 @@
 			<div class="jotplugins">
 				{{$jotplugins nofilter}}
 			</div>
+
+			{{if $scheduled_at}}{{$scheduled_at nofilter}}{{/if}}
```

Comment:
Is it an optional field?

Reply:
I had been unsure if possibly this could be something that could be made optional in the end.